packages/notifications/twillio-sms/src/schema/config.ts

```diff
@@ -0,0 +1,6 @@
+import isMobilephone from "validator/lib/isMobilePhone";
+import { z } from "zod";
+
+export const SmsConfigurationSchema = z.object({
+  phoneNumber: z.string().refine(isMobilephone),
```

Comment:
Can we use regex instead of adding a deps for that specific use case?

```js
const phoneRegex = new RegExp(
  /^([+]?[\s0-9]+)?(\d{3}|[(]?[0-9]+[)])?([-]?[\s]?[0-9])+$/
);

const schema = z.object({
     phone: z.string().regex(phoneRegex, 'Invalid Number!'),
})
```